third_party/airflow/tests/unit/test_airflow_operator_mock.py

```diff
@@ -69,65 +71,19 @@ def sleep_job():
         ],
     )
     return [submit_pb2.JobSubmitRequestItem(priority=0, pod_spec=pod)]
-
-
-def test_job_service_health():
```

Comment:
Why'd a bunch of tests disappear? They seem unrelated to your change.

Reply:
hmm that's not right -- fixing

Reply:
errant deletion, fixed